system/View/Parser.php

```diff
@@ -425,6 +430,58 @@ protected function parseComments(string $template): string
 	{
 		return preg_replace('/\{#.*?#\}/s', '', $template);
 	}
+	
+	/**
+	 * Replaces html colors with temporary value so it does not affect parsing regex
+	 * 
+	 * Takes ..color:#fff, background-color:#993399 and color=#0284D0> 
+	 * Returns ..color{{color:fff}}, background-color{{color:993399}}, and color{{color=0284D0}}
+	 * 
+	 * @param string $template
+	 * 
+	 * @return string
+	 */
+	function parseHtmlColors(string $template) : string
+	{
+		preg_match_all("/(:|=)(| )#(?:[a-f0-9]{3}|[a-f0-9]{6})\b/uim", $template, $matches);
+		if (!empty($matches)) {
+			foreach ($matches['0'] as $match) {
+				if (trim($match) != "") {
+					$template = str_replace($match, "{{color" . str_replace("#", "", $match) . "}}", $template);
+				}
+			}
+		}
+		return $template;
+	}
+
+	/**
+	 * Restores html colors to their original state
+	 *
+	 * @param string $parsedTemplate
+	 * @return string
+	 */
+	function unParseHtmlColors(string $parsedTemplate) : string
```

Comment:
please add modifier, you can use private for it: 

```suggestion
	private function unParseHtmlColors(string $parsedTemplate): string
```